### DISCUSSION TITLE
RAN Slicing Capabilities

### DISCUSSION THREAD

**Discussion by gab-arrobo** [srsran/srsRAN_Project#355] [General Help] [answered]
Hello everyone,

I was looking for the details about the RAN Slicing capabilities currently support by the project, but I did not find that many details in the documentation. So, I was wondering if you could please help me providing details about what is currently supported? Based on the documentation and the source code, I see that the KPM and RC service models are supported. Is there a plan to add the E2SM-CCC service model, which is used to control policy ratios for the different slices?

I would appreciate your input.

Thanks!

**Comment by yagoda** [comment, marked as answer]
Hi,

Slicing has not yet been introduced at RAN/scheduler level. NSSAI slicing in the NGAP, you can specify to the core what SSTs are supported by the gnodeb.  There will eventually be slicing support in the scheduler and the ability to change scheduling policies by slice using the RC service model.(There is limited support for this in the RC service model) 
The E2SM-CCC will eventually be added but it is a newer service model and not many 3rd party RIC providers are considering it yet which makes it more challenging for us to add.

Thanks,
Justin

**Reply by improperaffo** [reply]
Hi Justin. Is there a timeline for the slicing support in the scheduler?

Thanks
Raphael

**Comment by gab-arrobo** [comment]
Thanks @yagoda for your feedback. What is the typical process you guys follow to add/create/modify a service model?

**Reply by pgawlowicz** [reply]
@gab-arrobo Not sure if I understand your question correctly, but we are currently developing E2SM_KPM and E2SM_RC service modules. Both are already working with two RICs (Flexric and ORAN SC RIC), we are now adding and exposing more metrics to be monitored  (in KPM) and parameters to be changed (RC).

**Reply by gab-arrobo** [reply]
Hi @pgawlowicz, are you guys planning to add the E2SM_CCC service model or other service models? I am asking this because the E2SM_CCC service model is supposed to be used for RAN slicing management/control

**Reply by pgawlowicz** [reply]
Yes, eventually, we will add it, but there is no timeline for this task yet.

**Reply by gab-arrobo** [reply]
Ok, thanks!
